tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -211,22 +210,36 @@ spec:
 
     - name: report
       image: quay.io/enterprise-contract/ec-cli:snapshot
-      command: [cat]
```

Comment:
Could've used `onError: continue` here and the other steps as well. Maybe a little bit cleaner?